src/App.js

```diff
@@ -1,10 +1,21 @@
+import { BrowserRouter as Router, Route, Routes } from 'react-router-dom';
 import './App.css';
+import Navbar from './components/Navbar';
+import Home from './components/Home';
 import Calculator from './components/Calculator';
+import Quotes from './components/Quotes';
 
 function App() {
```

Comment:
- [ ] Greate job implementing the App 👍🏼, however, the design should be as close to the wireframe as possible. you can customize it but try to stick to the general layout, kindly consider moving the nav links to the right and adding some padding and margin to the pages so it takes advantage of the white space, also please you can adjust the calculator page so it looks like the Wireframe

![Screenshot_20220705_205218](https://user-images.githubusercontent.com/25227022/177405836-fa535487-bb59-4227-bee5-e9db79abb4df.png)